src/main/java/sh/platform/config/InfluxDB.java

```diff
@@ -0,0 +1,46 @@
+package sh.platform.config;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+import static org.influxdb.InfluxDBFactory.connect;
+
+/**
+ * A credential specialization that provides a {@link org.influxdb.InfluxDB}
+ */
+public class InfluxDB extends Credential implements Supplier<org.influxdb.InfluxDB> {
+
+    private static final String URL = "http://%s:%d";
+
+    public InfluxDB(Map<String, Object> config) {
+        super(config);
+    }
+
+    @Override
+    public org.influxdb.InfluxDB get() {
```

Comment:
Why is this full class name used inline rather than via an import, like everything else?

Reply:
We have InfluxDB as our class and the library use this as well, so I need to define which one I would like to use explicitly.